### ISSUE TITLE
[TalkBack] Phone number formatter speaks "end" instead of digit

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#203]
```
Steps to reproduce on cm10.1 on mako : 
note : this is in the U.S. Where the phone numbers look like (xxx)xxx-xxxx
1 - open phone
2 - dial three digits
3 - you'll hear "end"
4 - dial three more digits
5 - you'll hear "end"
6 - dial four more digits
7 - you'll hear end
what happens : see above
what should happen instead : "end" should not be announced.

```

Original issue reported on code.google.com by `billkd2008@gmail.com` on 11 Mar 2013 at 11:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 2 Apr 2013 at 10:40
- Changed title: **[TalkBack] Phone number formatter speaks "end" instead of digit**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v3.3.1_b1.
```

Original comment by `alanv@google.com` on 3 Apr 2013 at 1:00
- Changed state: **Fixed**
